docs/articles/documentation/guides/basic-guides/interact-with-the-page.md

```diff
@@ -0,0 +1,307 @@
+        .pressKey('space')
+});
+```
+
+## Navigate
+
+[Navigate](../../reference/test-api/testcontroller/navigateto.md) action navigates to the specified URL.
+
+**Example**
+
+```js
+import { Selector } from 'testcafe';
+
+fixture `Example`
+    .page `https://devexpress.github.io/testcafe/example/`;
+
+test('Navigate To URL test', async t => {
+    await t
+        .navigateTo('https://github.com/DevExpress/testcafe');
+});
+```
+
+## Type Text
+
+[Type Text](../../reference/test-api/testcontroller/typetext.md) action types the specified text into the selected input element.
+
+**Example**
+
+```js
+import { Selector } from 'testcafe';
+
+fixture `Example`
+    .page `https://devexpress.github.io/testcafe/example/`;
+
+test('Type Text test', async t => {
+    await t
+        .typeText('#developer-name', 'John Doe');
+});
+```
+
+## Select Text
+
+Actions that allow you to select text within inputs, `<textarea>` and `contentEditable` elements.
+
+* [Select Text in Input Elements](../../reference/test-api/testcontroller/selecttext.md)
+* [Select \<textarea\> Content](../../reference/test-api/testcontroller/selecttextareacontent.md)
+* [Perform Selection within Editable Content](../../reference/test-api/testcontroller/selecteditablecontent.md)
+
+**Example**
+
+```js
+import { Selector } from 'testcafe';
+
+fixture `Example`
+    .page `https://devexpress.github.io/testcafe/example/`;
+
+test('Select Text test', async t => {
+    await t
+        .typeText('#developer-name', 'John Doe')
+        .selectText('#developer-name')
+        .pressKey('delete');
+});
+```
+
+## Hover
+
+[Hover](../../reference/test-api/testcontroller/hover.md) action allows to hover over the tested page.
+
+**Example**
+
+```js
+import { Selector } from 'testcafe';
+
+fixture `Example`
+    .page `https://js.devexpress.com`;
+
+test('Hover test', async t => {
+    await t
+        .hover('.map-container');
+});
+```
+
+## Drag Elements
+
+**Drag** actions allow you to drag elements on the tested page.
+
+* [Drag an Element by an Offset](../../reference/test-api/testcontroller/drag.md)
+* [Drag an Element onto Another One](../../reference/test-api/testcontroller/dragtoelement.md)
+
+> The **drag** actions do not invoke integrated browser actions such as selecting text.
+> Use them to perform drag-and-drop actions that are processed by webpage elements, not the browser.
+> To select text, use [t.selectText](../../reference/test-api/testcontroller/selecttext.md).
+
+**Example**
+
+```js
+import { Selector } from 'testcafe';
+
+fixture `Example`
+    .page `https://devexpress.github.io/testcafe/example/`;
+
+test('Drag test', async t => {
+    const triedCheckbox = Selector('label').withText('I have tried TestCafe');
+
+    await t
+        .click(triedCheckbox)
+        .drag('#slider', 360, 0, { offsetX: 10, offsetY: 10 });
+});
+```
+
+## Upload Files
+
+Actions that allow you to work with file upload input elements.
```

Comment:
What do you mean by " to work with"?

Reply:
@dirk-pieterse `interact`, I'll rephrase this